src/main/java/net/minecraftforge/event/entity/EntityTeleportEvent.java

```diff
@@ -166,7 +166,11 @@ public EnderPearl(ServerPlayer entity, double targetX, double targetY, double ta
             this.hitResult = hitResult;
         }
 
+        /**
+         * @deprecated Use {@link #EnderPearl(ServerPlayer, double, double, double, ThrownEnderpearl, float, HitResult)} the hit result-sensitive version}
```

Comment:
_Technically_, this should use `{@linkplain}`.
```suggestion
         * @deprecated Use {@linkplain #EnderPearl(ServerPlayer, double, double, double, ThrownEnderpearl, float, HitResult)} the hit result-sensitive version}.
```